src/app/components/media/WebPageMediaCard.js

```diff
@@ -15,7 +15,7 @@ class WebPageMediaCard extends Component {
     const { projectMedia: { team }, projectMedia: { media: { metadata } } } = this.props;
     const embed = metadata;
     if (!embed.html) return false;
```

Comment:
@amoedoamorim the other Sentry error (https://meedan.sentry.io/issues/5687298353/events/b3a8bfe41a574ab7be310d120cce8e9b/) was in this line :thinking:

Reply:
hmmm, yes, investigating that

Reply:
I'm attributing that to bad source mapping in firefox. I've inspected the minified source served by the app and it's really team (mapped to 'e') that is null.

![image](https://github.com/user-attachments/assets/187e1e2c-cc46-49d9-8b6b-a5fe866af67d)

Reply:
![image](https://github.com/user-attachments/assets/38b4afaf-f598-43a5-9393-a765cd87c4b9)

Reply:
I also checked locally when reproducing the issue that `embed.html`  is valid content, not null